### ISSUE TITLE
GitCommit in commands package not being updated at build time

### ISSUE THREAD

**Issue by johnmccabe** [alexellis/faas-cli#62]
## Expected Behaviour

This results in the following output for a brew installed CLI:
```
$ ./faas-cli version
Git Commit: thisisavalidsha
```

## Current Behaviour

Following the merge of #50 and the move of the `GitCommit` variable from the old `main` package rather than `commands` package this commit SHA isn't getting populated.

This results in the following output:
```
$ ./faas-cli version
Git Commit:
```

## Steps to Reproduce (for bugs)

1. Download the current `0.4.9` release, or build locally using `build_redist.sh` and run `./faas-cli version`

## Your Environment

* Docker version `docker version` (e.g. Docker 17.0.05 ):
n/a

* Are you using Docker Swarm or Kubernetes (FaaS-netes)?
n/a

* Operating System and version (e.g. Linux, Windows, MacOS):
MacOS

* Link to your project or a code example to reproduce issue:
n/a

**Comment by johnmccabe** [comment]
spotted by @ems5311 in the #faas channel on Slack

**Comment by ems5311** [comment]
I've created a pull request with a fix for this bug here:

#65

**Comment by alexellis** [comment]
Fixed by #62.
